api/api.go

```diff
@@ -82,27 +88,43 @@ func Setup(ctx context.Context, r *mux.Router, cognitoClient cognito.Client, use
 	}
 
 	r.HandleFunc("/v1/tokens", contextAndErrors(api.TokensHandler)).Methods(http.MethodPost)
-	r.HandleFunc("/v1/tokens", contextAndErrors(api.SignOutAllUsersHandler)).Methods(http.MethodDelete)
+	r.HandleFunc("/v1/tokens", auth.Require("users:update", contextAndErrors(api.SignOutAllUsersHandler))).
```

Comment:
Rather than inline, I'd create an enumeration in the respective model package (`models/users.go` and `models/groups.go`) and reference those in the respective handlers:

```suggestion

      // authorisation permissions
      const (
        UsersRead string = "users:read"
	UsersCreate         = "users:create"
	UsersUpdate        = "users:update"
       )


	r.HandleFunc("/v1/tokens", auth.Require(models.UsersUpdate, contextAndErrors(api.SignOutAllUsersHandler))).
```